requirements/base.txt

```diff
@@ -1,8 +1,9 @@
 # Requirements for app run
 
-git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
-django-statici18n==1.8.0
-edx-i18n-tools==0.4.7
-Mako==1.0.7
-bleach==2.1.4
-html-xblock~=0.1.1
+git+https://github.com/edx/xblock-utils.git@2.1.1#egg=xblock-utils==2.1.1
+django-statici18n==1.9.0
+edx-i18n-tools==0.5.3
+Mako==1.1.3
+bleach==3.1.5
+-e git+https://github.com/open-craft/xblock-html@taniwha/bb-2659_Juniper_xblock-html_and_xblock-completable#egg=html-xblock==0.2.0
```

Comment:
@taniwha, this has to be updated to `html-xblock==<version>` once the related `xblock-html` PR is merged, a release tagged and deployed to PyPI.